iron-a11y-keys-behavior.html

```diff
@@ -152,7 +104,7 @@
       var validKey = '';
       if (keyIdent) {
         if (IDENT_CHAR.test(keyIdent)) {
-          validKey = KEY_IDENTIFIER[keyIdent];
+          validKey = KEY_IDENTIFIER[keyIdent] || String.fromCharCode(parseInt(keyIdent.replace('U+', '0x'), 16)).toLowerCase();
```

Comment:
rely on fromCharCode to retrieve the corresponding symbol. The final toLowerCase will handle capitalized letters 
KEY_IDENTIFIER is now used only for special characters like 'esc', 'backspace', etc..